reframe/core/pipeline.py

```diff
@@ -132,6 +132,25 @@ class RegressionTest(RegressionMixin, jsonext.JSONSerializable):
     This class provides the implementation of the pipeline phases that the
     regression test goes through during its lifetime.
 
+    :param require_version: is a list of ReFrame version specifications that
```

Comment:
We should make explicit that these parameters are set at the *class definition* (not instantiation). The Python documentation also uses that term to classify such parameters.